pages/assessment.tsx

```diff
@@ -121,38 +152,20 @@ const Assessment: NextPage = () => {
               Description={markersGuideDescription}
             />
           </div>
-          {/* Display two answers next to eachother when max items is four or less  */}
-          {maxItemsPerPage <= 4 ? (
-            <div className={styles.grid4answers}>
-              {assessments
-                .slice(
-                  currentPage * maxItemsPerPage - maxItemsPerPage,
-                  currentPage * maxItemsPerPage
-                )
-                .map((assessment: AssessmentType) => (
-                  <Textbox
-                    key={assessment.assessmentId}
-                    assessment={assessment}
-                    setAssessment={setAssessment}
-                  />
-                ))}
-            </div>
-          ) : (
-            <div className={styles.grid}>
-              {assessments
-                .slice(
-                  currentPage * maxItemsPerPage - maxItemsPerPage,
-                  currentPage * maxItemsPerPage
-                )
-                .map((assessment: AssessmentType) => (
-                  <Textbox
-                    key={assessment.assessmentId}
-                    assessment={assessment}
-                    setAssessment={setAssessment}
-                  />
-                ))}
-            </div>
-          )}
+          <div className={styles.grid}>
+            {assessments
+              .slice(
+                currentPage * maxItemsPerPage - maxItemsPerPage,
+                currentPage * maxItemsPerPage
+              )
+              .map((assessment: AssessmentType) => (
+                <Textbox
+                  key={assessment.assessmentId}
+                  assessment={assessment}
+                  setAssessment={setAssessment}
+                />
+              ))}
```

Comment:
This is a "kinkig" problem, but if you use the standard setting of 4 assessments per page, then assess eg. 16 assessments, and after, you click next and want to change the number of items per page. You then get two situations:
* change to 2 items per page

Then the assessments are already assessed when you click next
* change view to 6 items per page

Then there assessments that are not assessed earlier (if you click back, all of the assessments are not assessed)

Reply:
So I'm thinking that you should go tho the lowest page number where you have un-assessed answers when you change view. 
I also think it i most important to avoid the 4-view --> 6-view issue.